src/components/UserStats/IncomeMenuItem.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <ul
+    :class="[
+      'flex',
+      'justify-center',
+      'items-center',
+      'bg-shade-gray',
+
+      'rounded-[58px]',
+      'p-[4px]',
+      'overflow-hidden',
+    ]"
+  >
+    <li
+      v-for="item in typeMenuItemList"
+      :key="item.value"
+      class="flex items-center justify-center w-full"
+    >
+      <div
+        :class="[
+          'flex',
+          'flex-col',
+          'justify-center',
+          'items-center',
+
+          'w-full',
+          'py-[8px]',
+          'rounded-[58px]',
+          'cursor-pointer',
+          'transition-all',
+          'duration-200',
+
+          'text-dark-gray bg-shade-gray hover:text-medium-gray',
+          { '!bg-like-green text-shade-gray shadow-sm hover:text-like-cyan-gray': item.isSelected },
+
+        ]"
+        @click="$emit('on-type-click',item.value)"
```

Comment:
Vue typically event emitter normally not start with `on` like React 
For example, `onClick` -> `click`